File: Baekjoon/Class2/05.2292.java
import java.io.*;

class Main {
    public static void main(String[] args) throws IOException {
        int n, lastRoom = 1, dist = 1;
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        n = Integer.parseInt(br.readLine());
        while(n > lastRoom) {
            lastRoom += dist * 6; // 마지막 방 번호가 i * 6씩 증가함
            dist++;
        }
        System.out.print(dist);
        br.close();
    }
}

File: Baekjoon/Class2/05.2292.py
import sys
input = sys.stdin.readline

n = int(input())
last_room = 1
dist = 1
while n > last_room:
    last_room += dist * 6 #마지막 방 번호가 i * 6씩 증가함
    dist += 1
print(dist)